tools/challenge-editor/api/configs/super-block-list.ts

```diff
@@ -78,5 +78,9 @@ export const superBlockList = [
   {
     name: 'A2 English for Developers (Beta)',
     path: '21-a2-english-for-developers'
+  },
+  {
+    name: 'Rosetta Code',
+    path: '22-rosetta-code'
```

Comment:
Same here 😃 

It looks like this will need to be updated to something other than 22 since Tom is also using that the Python curriculum

https://github.com/freeCodeCamp/freeCodeCamp/pull/52628/files#diff-7dc5a3c37a34a8e58cef2f545da6da29e0b376364c3c16a85b4a46c83e7baf4f